openfoam-interface/internal-interface/libs/sourceFiles/LESModel.C

```diff
@@ -0,0 +1,210 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
There were no differences between this file `LESModel.C` and the one found in OpenFoam 2.2.x ("OpenFOAM-XXX/src/turbulenceModels/incompressible/LES/LESModel/LESModel.C"). Can this file be removed and the one from OpenFoam used?